chevah/keycert/ssh.py

```diff
@@ -676,7 +676,7 @@ def _fromString_PRIVATE_BLOB(cls, blob):
             dsakey = cls(DSA.construct((y, g, p, q, x)))
             return dsakey
         else:
-            raise BadKeyError('unknown blob type: %s' % keyType)
+            raise BadKeyError('Unknown blob type: %s.' % keyType)
```

Comment:
I think that here we should use %r to make sure we don't try to decode random data.... and this should be valid for any other exception in which we inject a non-sanitized input